### ISSUE TITLE
Conditional hidden required fields

### ISSUE THREAD

**Issue by klevron** [verbb/formie#648]
**Description**

Conditionally hidden required field is still required on server side :

![image](https://user-images.githubusercontent.com/4560725/142601074-50be55dc-5237-4ca8-b7cf-8968b04f2161.png)

**Steps to reproduce**
1. create a form with a conditional hidden required field
2. submit the form when the field is hidden

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.25
- Craft version: 3.7.20
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Are you able to let me know the condition settings for Message, and what sort of field Message is? Any field that's deemed as conditionally hidden should be excluded from validation.

**Comment by klevron** [comment]
It is just a message field from the default contact form :
![image](https://user-images.githubusercontent.com/4560725/142604959-57df54b4-6d08-411f-97e9-82d87db79a6b.png)

**Comment by engram-design** [comment]
Ah, I see. Looks like there was an issue when evaluating empty strings, where we strip them out too early when protecting against empty conditions.

Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.25",
  "...": "..."
}
```

Then run `composer update`.

**Comment by klevron** [comment]
It works, thanks for this very fast answer and fix 👍

**Comment by engram-design** [comment]
Nice one! Will keep open until I’ve pushed this in a release.

**Comment by engram-design** [comment]
Fixed in [1.4.26](https://github.com/verbb/formie/releases/tag/1.4.26)
